continuous_integration/environment-3.8-jdk11-dev.yaml

```
@@ -31,13 +31,5 @@ dependencies:
 - tpot
 - tzlocal=2.1
 - uvicorn=0.11.3
-# fugue dependencies; remove when we conda install fugue
-- adagio
-- antlr4-python3-runtime<4.10
-- ciso8601
-- fs
-- pip
-- qpd
-- triad
 - pip:
-  - fugue[sql]==0.5.3
+  - fugue>=0.7.0
```

The python 3.8 testing environments stay pinned to our minimum supported version to make sure it still works as expected - so depending on what we want our minimum Fugue dependency to be, this would need to be changed to:

```suggestion
  - fugue==0.7.0
```

---

Sorry, I need to make the same change to 3.8